### ISSUE TITLE
Unable to Deploy to Static Website

### ISSUE THREAD

**Issue by alexsilvar** [microsoft/vscode-azurestorage#871]
Does this occur consistently? Yes
Repro steps:
1. Click on Deploy to Static Website
2. See the error

Action: azureStorage.deployStaticWebsite
Error type: UNABLE_TO_GET_ISSUER_CERT
Error Message: request to https://pnsportaldev.blob.core.windows.net/?restype=service&comp=properties failed, reason: unable to get issuer certificate

Version: 0.10.1
OS: win32
OS Release: 10.0.18363
Product: Visual Studio Code
Product Version: 1.50.1
Language: en

<details>
<summary>Call Stack</summary>

```
ClientRequest.<anonymous> extension.bundle.js:166:16389
ClientRequest.emit events.js:228:7
TLSSocket.socketErrorListener _http_client.js:406:9
TLSSocket.emit events.js:223:5
emitErrorNT destroy.js:92:8
emitErrorAndCloseNT destroy.js:60:3
processTicksAndRejections task_queues.js:81:21
```

</details>

**Comment by nturinski** [comment]
Could you provide detailed repro steps and the full error logs for this?

This might be related to your issue: https://stackoverflow.com/questions/36494336/npm-install-error-unable-to-get-local-issuer-certificate

Seems like you might have to set up your proxy and that a npm package that is being installed is throwing this error.

**Comment by alexsilvar** [comment]
Hey @nturinski you can check the Call Stack dropdown.
I got this error while trying to deploy the application using the Azure Storage Extension.

I tried to use ``` npm config set strict-ssl false``` as the post you shared suggested with no luck.

This feature worked previously (about a month ago) and last week when I needed to use it again, it started to fail. The manual deploy on Azure blob storage is a pain and I don't have the CI/CD for my application yet.

This could be related to my .npmrc file? I use a proxy inside my organization for it.

**Comment by nturinski** [comment]
The call stack dropdown doesn't give me the information from your output channel which is what I'm interested in seeing.  Because the error seems like it's happening server/Azure side, I was wondering if there was any other helpful/insightful output leading up to that error.

It's definitely possible.  Have you changed your .npmrc file within the last month?  

We also released v0.10.0 back in September, so maybe it's related to the new update?  You could try downgrading to v0.9.0 and see if that resolves your issue.
